testing/stress_test.py

```diff
@@ -209,8 +209,6 @@ def shutdown(sig, frame):
 
 if __name__ == "__main__":
     signal.signal(signal.SIGINT, shutdown)
-    # set up logging
-    logging.basicConfig(level=logging.DEBUG, format="%(name)-12s %(levelname)-8s: %(message)s")
```

Comment:
with this removed,  logging shows up  for the simulator, but not for the  test

Reply:
@plarios The latest should fix this, I set the default logging handler as well as the categories now:
https://github.com/monoDriveIO/client/commit/76a24c87284af7fa10480a963a489d26b4165ef2